src/Symfony/Bundle/FrameworkBundle/DependencyInjection/Configuration.php

```diff
@@ -984,6 +984,12 @@ private function addMessengerSection(ArrayNodeDefinition $rootNode)
                                 ->end()
                             ->end()
                         ->end()
+                        ->arrayNode('doctrine_transaction')
+                            ->canBeEnabled()
+                            ->children()
+                                ->scalarNode('entity_manager_name')->info('The name of the entity manager to use')->defaultValue('default')->end()
```

Comment:
Could we make it `null` by default and let Doctrine determine the default name? ([see](https://github.com/doctrine/common/blob/a61bfddd39b887265117806e81cc6ca8da242fe1/lib/Doctrine/Common/Persistence/AbstractManagerRegistry.php#L157-L159))

Reply:
Excellent idea. Thanks